README.md

```diff
@@ -84,33 +87,35 @@ export JAVA_HOME=$(/usr/libexec/java_home -v 13)
 
 ### **Building and Running the Playground App**
 
-On either Mac or Windows, NPX is used to build and run the Playground sample/test app from the command line. Open a command prompt at the root of the BabylonReactNative repo if you don't have one already open.
+On either Mac or Windows, NPM is used to build and run the Playground sample/test app from the command line. Open a command prompt at the root of the BabylonReactNative repo if you don't have one already open.
 
 #### Android
 
 ```
 cd Apps/Playground
-npx react-native run-android
+npm run android
```

Comment:
When setting up the demo, I ran into problems doing this, probably because I don't have `react-native-cli` globally installed. To solve those problems and run the demo successfully, I had to blast the repo and go back to using the `npx` command.

Reply:
I don't believe you should need `react-native-cli` globally installed. Do you mean you ran into problems specifically using the command `npm run android`?

Reply:
*Hmm*, actually, upon review, I think I ran into problems with `react-native run-android`, not `npm run android`. Sorry about the confusion, my mistake.